glut/Makefile.am

```diff
@@ -67,8 +67,7 @@ endif
 
 test_glut_SOURCES=test_glut.c
 test_glut_LDADD=$(addprefix $(top_builddir)/, $(LIBS_MGH)) $(LIBS_VXL) -lz \
-	$(LIB_JPEG) $(LIB_TIFF)	$(X_LIBS) $(LIBS_OPENGL) $(top_builddir)/glut/libglut.a
-
+	$(LIB_JPEG) $(LIB_TIFF)	$(top_builddir)/glut/libglut.a $(X_LIBS) $(LIBS_OPENGL) 
```

Comment:
Many of the test scripts check for an environment variable which, if defined, will skip the test. For example, SKIP_MRI_CA_LABEL_TEST, SKIP_MR_CA_REGISTER_TEST, (and others) will skip those respective tests:

https://github.com/freesurfer/freesurfer/blob/dev/mri_ca_label/test_mri_ca_label

I suppose this could be generalized in some way to produce a subset of tests. But am I correct that we are limited to only 50 minutes. Perhaps some trickery can be used to get around this.